src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -101,6 +106,14 @@ private static Point GetAnimationAnchor(Point origin, Size size, double centerX,
 					? origin.Y
 					: centerY / size.Height + origin.Y);
 
+		static GPUFloatValueAnimator()
+		{
+			if (CoreWindow.GetForCurrentThread() is { } coreWindow)
```

Comment:
You may want to register on the first instance, `GetForCurrentThread()` may be executed at unwanted times :)